CMakeLists.txt

```diff
@@ -76,6 +76,15 @@ cmake_dependent_option(BUILD_LIBTAR "Build with support for libtar library" ON
                        "NOT BUILD_DALI_NODEPS" OFF)
 option(BUILD_FFTS "Build with ffts support" ON)  # Built from thirdparty sources
 
+set(CUSTOM_KERNEL_SRCS_PATTERN "" CACHE STRING
+    "Glob expression to build a subset of files for the kernel lib")
+set(CUSTOM_KERNEL_TEST_SRCS_PATTERN "" CACHE STRING
```

Comment:
How about having `CUSTOM_SRCS_PATTERN`, that filters all of the targets? This way I would need to know, which file belongs to which target.

Reply:
Would not work. I do `*slice*` and now only files matching that will get build. Build won't succeed. We need a lot of core utilities, pipeline, executor, etc... It wouldn't be practical for the user to list everything.